department-of-veterans-affairs/developer-portal
# Update CodeBuild deploy to deploy from the main release without the "cb-" prefix

### Description



this PR updates the CodeBuild deploy to deploy from the main release as opposed to the secondary one with a "cb-" prefix. currently, CodeBuild creates a release in Github with a tag of the form `cb-developer-portal/v0.0.0`, and Jenkins creates the main release with a tag of the form `developer-portal/v0.0.0`. so, then, this PR is updating the CodeBuild deploy to use the release that is currently created by Jenkins but will be created by CodeBuild in the near future.

note that this deploy job does not run automatically anywhere right now. this PR will allow us to manually test the deploy job in dev and staging from a commit hash and from a release. once we have confidence in the deploy job, we will then cut over the dev and staging deploys that run on merges to master to be run from CodeBuild instead of Jenkins.

this PR also turns on PR comments in CodeBuild in preparation for the cutover. this change will temporarily cause redundant comments (roughly) from CodeBuild and Jenkins.

### Process



- [n/a] Tests added or updated for change
  - manually testing deploys  
- [n/a] Docs updated
  - [n/a] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [n/a] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [n/a] Accessibility concerns have been considered and addressed
  
  - [n/a] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### Requested feedback
